src/scss/modules/_typography.scss

```diff
@@ -0,0 +1,12 @@
+@import url('https://fonts.googleapis.com/css2?family=Barlow+Semi+Condensed:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+$ff: 'Barlow Semi Condensed', sans-serif;
+
```

Comment:
The typography of this app looks great as is, however, as per the design  requirements of this project : -

- [ ] The Typography: font face, size, and weight should be lato and gillsans as mentioned in the style guidelines.